tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -25,10 +25,11 @@ Vue.use(Vuelidate)
 const vuetify = new Vuetify({})
 const store = getVuexStore() as any // remove typings for unit tests
 
-// Boilerplate to prevent the complaint "[Vuetify] Unable to locate target [data-app]"
-const app: HTMLDivElement = document.createElement('div')
-app.setAttribute('data-app', 'true')
-document.body.append(app)
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+
+// Prevent the warning "[Vuetify] Unable to locate target #filing-history-list"
+document.body.setAttribute('id', 'filing-history-list')
```

Comment:
This adds an id to the mounted object so that the dialog can attach itself to it.

Reply:
@cameron-freshworks FYI, to preempt the warning for dialogs attachments ^^